wgpu-native/cbindgen.toml

```diff
@@ -1,4 +1,7 @@
-header = "#define WGPU_LOCAL"
+header = """
```

Comment:
Alternatively we could just add `"wgpu_instance_create_surface_from_winit"` to `exclude` to remove this from the header